pom.xml

```diff
@@ -28,9 +28,10 @@
     <version>40</version>
   </parent>
 
+  <!-- updated ztesler, 04-2016 -->
   <groupId>org.apereo.application</groupId>
   <artifactId>ImageQuiz</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>2.0</version>
```

Comment:
Following semantic versioning, this is likely 1.0.1 (if only patches) or 1.1.0 if there are new features. Since it looks compatible with previous versions, it really does not look like a 2.0 release.

Thanks Christian for the review!